themes/light/twine.css

```diff
@@ -1,4 +1,5 @@
-/* Legacy theme, do not use as example. */
+/* [THEME META] {"ver":1, "author":"SYSTEM", "description":"Simple and unique light theme.", "accessibility":"ignore"} */
+/*Legacy theme - do not use as example*/
```

Comment:
Tiny bit of pedantry but do you think there should be a newline between the above and the content to maintain consistency or nah?